azure/pas-single-component-deploy-in-pas-hybrid-network.json

```diff
@@ -162,7 +163,7 @@
                         "value": "[parameters('Vault Private IP')]"
                     },
                     "Guid": {
-                        "value": "[reference('GuidGenerator').outputs.guid.value]"
+                        "value": "pid-920ca5c9-254e-5fa2-87f3-eba1b62dc92d"
```

Comment:
I think that second PSMP will not work now because it is using guid in order to generate user.